src-tauri/Cargo.toml

```diff
@@ -14,20 +14,27 @@ rust-version = "1.75"
 tauri-build = { version = "1.2", features = [] }
 
 [dependencies]
+keyring = "2"
+log = "^0.4"
+machine-uid = "0.2"
+nom = "7"
+notify = "6"
+oauth2 = "4"
+regex = "1"
+reqwest = { version = "0.11", features = ["json", "multipart"] }
```

Comment:
Tauri includes an HTTP library based on Reqwest but it's less feature-complete than this version (namely it lacks built-in JSON handling). Could go either way here but this was a bit easier.